website/pages/use-cases/simple-container-orchestration.jsx

```diff
@@ -2,8 +2,29 @@ import UseCasesLayout from 'components/use-case-page'
 import TextSplitWithCode from '@hashicorp/react-text-split-with-code'
 import TextSplitWithImage from '@hashicorp/react-text-split-with-image'
 import FeaturedSliderSection from 'components/featured-slider-section'
+// Imports below are used in getStaticProps only
+import highlightData from '@hashicorp/nextjs-scripts/prism/highlight-data'
 
-export default function SimpleContainerOrchestrationPage() {
+export async function getStaticProps() {
+  const codeBlocksRaw = {
+    containerOrchestration: {
+      code:
+        'task "webservice" {\n  driver = "docker"\n\n  config {\n    image = "redis:3.2"\n    labels {\n      group = "webservice-cache"\n    }\n  }\n}',
+      language: 'hcl',
+    },
+    windowsSupport: {
+      code:
+        'sc.exe start "Nomad"\n\nSERVICE_NAME: Nomad\n      TYPE               : 10  WIN32_OWN_PROCESS\n      STATE              : 4  RUNNING\n                              (STOPPABLE, NOT_PAUSABLE, ACCEPTS_SHUTDOWN)\n      WIN32_EXIT_CODE    : 0  (0x0)\n      SERVICE_EXIT_CODE  : 0  (0x0)\n      CHECKPOINT         : 0x0\n      WAIT_HINT          : 0x0\n      PID                : 8008\n      FLAGS              :',
+    },
+    multiRegionFederation: {
+      code: 'nomad server join 1.2.3.4:4648',
```

Comment:
I tried adding `shell-session` highlighting here to reflect the earlier appearance with the prefixed `$`, but the highlighting looked quirky on the address part of the command:

<img width="1317" alt="CleanShot 2021-01-07 at 11 56 41@2x" src="https://user-images.githubusercontent.com/4624598/103920687-97d24100-50df-11eb-927c-d24ff45417fd.png">

So I've left this snippet without highlighting for now, and have added [a task to our platform board to investigate `shell-session` highlighting](https://app.asana.com/0/1100423001970639/1199517937219949/f).